public/app/plugins/datasource/cloudwatch/components/ConfigEditor.tsx

```diff
@@ -227,7 +233,10 @@ export class ConfigEditor extends PureComponent<Props, State> {
           {options.jsonData.authType === 'arn' && (
             <div className="gf-form-inline">
               <div className="gf-form">
-                <InlineFormLabel className="width-14" tooltip="ARN of Assume Role">
+                <InlineFormLabel
+                  className="width-14"
+                  tooltip="ARN of role to assume. The AWS SDK Default credentials-chain will be used to assume it."
```

Comment:
Thanks for working on this! I'm a Grafana Cloud user and I'd like to be able to specify the credentials used when assuming the role, since the Grafana Cloud instances don't have IAM instance profiles (runs on GCP) and won't have default credentials configured (I don't have access to local disk/k8 to create `~/.aws/config` or to set env vars for AWS SDK).

Will it be possible to configure both a Role to assume & non-default creds to use when assuming it?

Reply:
Same question, as a user I expect to be able to specify aws_access_key_id, aws_secret_access_key and assumed role ARN through UI or API.

Reply:
This is something I did consider but in order to make it intuitive more frontend development was required than I am comfortable doing :sweat_smile:

Or at least in the way I originally thought about it. I just though of another solution that might work. I'll give it a go this in the coming days and see how it turns out.

Reply:
I just committed a change that moves `Assume Role ARN` *from* the `Authentication Provider` dropdown into its own textbox. 

This would then mean that the given `Authentication Provider` is used to assume the specified role (if a role is specified in the first place).

Does this solve your concerns or have I missed something? :)

Reply:
I think so ! Thanks for taking care of this :-)